app/helpers/path_finder_helper.py

```diff
@@ -19,3 +21,11 @@ def get_path_finder():
 
 
 path_finder = LocalProxy(get_path_finder)
+
+
+def requires_full_routing_path(function):
```

Comment:
Sorry to be a pain but I now realise that the convention seems to be to put `required` at the end e.g. `login_required` and `role_required` so this should probably be `full_routing_path_required`?

Reply:
@collisdigital good spot.